test-network-function/accesscontrol/suite.go

```diff
@@ -130,16 +156,103 @@ func runTestOnPods(env *config.TestEnvironment, testCmd testcases.BaseTestCase,
 	})
 }
 
+func getCrsNamespaces(crdName, crdKind string) map[string]string {
+	gomega.Expect(crdKind).NotTo(gomega.BeEmpty())
+	getCrNamespaceCommand := fmt.Sprintf(ocGetCrNamespaceFormat, crdKind)
+	cmdOut := utils.ExecuteCommand(getCrNamespaceCommand, common.DefaultTimeout, common.GetContext(), func() {
+		tcClaimLogPrintf("CRD %s: Failed to get CRs (kind=%s)", crdName, crdKind)
+	})
+	// No CRs created for this CRD yet.
+	if cmdOut == "" {
+		return map[string]string{}
+	}
+	lines := strings.Split(cmdOut, "\n")
+
+	crNamespaces := map[string]string{}
```

Comment:
move this to line 165, returning it as an empty list in line 167 or fill it with values in there's any